dev.html

```diff
@@ -2799,6 +2952,296 @@ <h2>Conditions</h2>
+                 - predicates: ex:accordingTo
+                   objects: http://example.org/person/$(Name)
+              student:
+                subject: http://example.org/student/$(Student)
+                predicateobjects:
+                 - [a, ex:Student]
+          </pre>
+        </div>
+        <div id="tab61-2">
+            <pre id="example-fn-oneline-rml-no-prefix" class="example" title="Quoted mapping in subject">
+              <#person>
+                a rr:TriplesMap ;
+                rml:subjectMap [
+                  rml:quotedTriplesMap <#student>;
+                ] ;
+                rr:predicateObjectMap [
+                  rr:predicate ex:accordingTo ;
+                  rml:objectMap [
+                    rr:template "http://example.org/person/{Name}";
+                  ];
+                ] .
+
+              <#student>
+                a rr:TriplesMap ;
+                rml:subjectMap [
+                  rr:template "http://example.org/student/{Student}";
+                ] ;
+                rr:predicateObjectMap [
+                  rr:predicate rdf:type ;
+                  rml:object ex:Student
+                ] .
+            </pre>
+        </div>
+      </div>
+
+      Quoted triples can be non asserted in the RDF-star graph using the <code>QuotedNonAsserted</code> key
+
+      <div id="tabs-62" class="nanotabs">
+        <ul>
+          <li><a href="#tab62-1" class="selected">YARRRML</a></li>
+          <li><a href="#tab62-2">RML</a></li>
+        </ul>
+        <div id="tab62-1">
+          <pre id="example-fn-oneline-no-prefix" class="example" title="Quoted non-asserted mapping in object">
+            mappings:
+              person:
+                subject: http://example.org/person/$(Name)
+                predicateobjects:
+                 - predicates: ex:confirms
+                   objects:
+                    - quotedNonAsserted: student
+              student:
+                subject: http://example.org/student/$(Student)
+                predicateobjects:
+                 - [a, ex:Student]
+          </pre>
+        </div>
+        <div id="tab62-2">
+            <pre id="example-fn-oneline-rml-no-prefix" class="example" title="Quoted non-asserted mapping in object">
+              <#person>
+                a rr:TriplesMap ;
+                rml:subjectMap [
+                  rr:template "http://example.com/person/{Name}"
+                ] ;
+                rr:predicateObjectMap [
+                  rr:predicate foaf:firstName ;
+                  rml:objectMap [
+                    rml:quotedTriplesMap <#student>;
+                  ];
+                ] .
+
+              <#student>
+                a rml:NonAssertedTriplesMap, rr:TriplesMap ;
+                rml:subjectMap [
+                  rr:template "http://example.org/student/{Student}";
+                ] ;
+                rr:predicateObjectMap [
+                  rr:predicate rdf:type ;
```

Comment:
Additional spaces, sometimes the `;` or `.` is without a space sometimes with.

Reply:
not relevant for this PR IMO, but good as general comment for the spec

Reply:
Yeah I just noticed it now, once we clean up the spec later, we need to pay attention to these, I made an issue: https://github.com/kg-construct/yarrrml-spec/issues/5